src/hotspot/share/memory/memoryReserver.cpp

```diff
@@ -437,7 +437,7 @@ ReservedSpace HeapReserver::Instance::try_reserve_range(char *highest_start,
 
   // Try attach points from top to bottom.
   for (char* attach_point = highest_start;
-       attach_point >= lowest_start && attach_point <= highest_start;  // Avoid wrap around.
+       attach_point >= lowest_start;  // Avoid wrap around.
```

Comment:
The comment is no longer true or relevant.  That comment was about the clause that is being removed.

Reply:
The unrelevant comment has been removed.